modules/custom/openy_activity_finder/config/install/openy_activity_finder.settings.yml

```diff
@@ -1,5 +1,6 @@
 backend: openy_activity_finder.solr_backend
+index: default
```

Comment:
Should we implement hook_update_N for those who already uses previous revision of this config?
I guess - yes. Could you make it, please?

Reply:
@podarok Fixed, tnx.